hack/development-mode.sh

```diff
@@ -2,19 +2,49 @@
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/.. 
 
-if [ -z "${MY_GIT_REPO_URL}" ]; then
-    MY_GIT_REPO_URL=$(git ls-remote --get-url | sed 's|^git@github.com:|https://github.com/|')
+if [ -f $ROOT/hack/development-mode.env ]; then
+    source $ROOT/hack/development-mode.env
 fi
-if [ -z "${MY_GIT_BRANCH}" ]; then
-    MY_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if [ -z "$MY_GIT_FORK_REMOTE" ]; then
+    echo "Set MY_GIT_FORK_REMOTE environment to name of your fork remote"
```

Comment:
Shouldn't we exit here or try to use existing remote if it is the only one?

Reply:
Right, thanks, added